docs/workloads/azure/data/etl_pipelines/silver_data_azure.md

```diff
@@ -12,10 +12,26 @@ keywords:
   - databricks
 ---
 
-Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data
-(e.g. CSV files) as an input and converts it into a more structured delta format.
-Azure Data Lake Storage (ADLS Gen2) has been chosen as both the source and destination system for
-our data processing.
+Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data of various formats
+(e.g. CSV, Parquet, JSON, Delta) as an input and converts it into a more structured Delta format.
```

Comment:
```suggestion
The Stacks data platform contains an example 'Silver' data processing workload, performing [Bronze to Silver layer](./etl_intro_data_azure.md#data-pipelines) transformations. It accepts raw data of various formats
(e.g. CSV, Parquet, JSON, Delta) as an input and converts it into a more structured Delta format.
```

Reply:
done, but without the reference link - I think there is no reason to go back to the introduction here.